### PR TITLE
Add warnign when trying to enable AutoDJ with left & right deck

### LINKED ISSUE #10903: Show warning when trying to enable AutoDJ if there's no left & right deck (xfader assign switches)
currently AutoDJ simply refuses to start (or stops if the xfader switches are flpped while AutoDj is running),

### PR DESCRIPTION
**Pull request by MetriSomesh**
Added warning message when trying to enable AutoDJ if there's no left & right deck.

### CODE REVIEW

**Comment by MetriSomesh** [comment]
@ronso0  Hello it is failing in 1 case what I have to do now?

**Comment by ronso0** [comment]
Thank you for working on this.
Please make yourself familiar with AutoDJ (to understand the case you are fixing) so you can improve the wording to be actually helpful for users.

**Comment by MetriSomesh** [comment]
Yeah I have read the manual and I understood why this issue was opened. But I don't understand why the pre-commit test case if failing

**Comment by daschuer** [comment]
When you click on "Details" you see a the pre-commit complains. 
clang-format wants the case labels aligned with the switch statement, so you need to reduce the indentation by 4 spaces. 
I think the original code is from our pre clang-format legacy.

Alternatively you can download and apply the Patch from the bottom of the summary page. 

The best of all options is to install pre-commit that these issues are fixed automatically during commit:
https://github.com/mixxxdj/mixxx/wiki/Using-Git#set-up-automatic-code-checking

File: src/library/autodj/dlgautodj.cpp [hunk 1]
```diff
@@ -273,6 +273,13 @@ void DlgAutoDJ::fadeNowButton(bool) {
 void DlgAutoDJ::toggleAutoDJButton(bool enable) {
     AutoDJProcessor::AutoDJError error = m_pAutoDJProcessor->toggleAutoDJ(enable);
     switch (error) {
+        case AutoDJProcessor::ADJ_NOT_TWO_DECKS:
+            QMessageBox::warning(nullptr,
+                tr("Auto DJ"),
+                tr("No deck assigned, Deck must be assigned to enable Auto DJ mode."),
```
Comment (daschuer) [inline src/library/autodj/dlgautodj.cpp:279]: How about this: 
"Two decks with opposite crossfader orientation are requires to enable Auto DJ mode."

Comment (ronso0) [reply src/library/autodj/dlgautodj.cpp:279]: or even shorter
"Auto DJ requires two decks assigned to opposite sides of the crossfader."

**Comment by ronso0** [comment]
@MetriSomesh Can you please rebase this onto the 2.3 branch, the force-push and choose 2.3 as base branch for this PR?
This way the change can slip in the next 2.3 bugfix release.

**Comment by MetriSomesh** [comment]
@ronso0 Ok I will rebase it on the 2.3 branch

**Comment by MetriSomesh** [comment]
@ronso0  I  rebase it on the 2.3 branch , but while doing the force-push it says Everything is up-to-date

**Comment by daschuer** [comment]
@MetriSomesh: see my comment here https://github.com/mixxxdj/mixxx/pull/11007#issuecomment-1291878916

**Pull request closed without merging**